client/reader/discover/follow-button.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FollowButton from 'reader/follow-button';
+import * as discoverStats from './stats';
+
+class DiscoverFollowButton extends React.Component {
```

Comment:
Just curious if we need the `class` here. Would this work as a _pure_ function or does `localize` only accept a class?